example/ios/Flutter/flutter_export_environment.sh

```diff
@@ -1,11 +1,15 @@
 #!/bin/sh
 # This is a generated file; do not edit or check into version control.
-export "FLUTTER_ROOT=C:\flutter\flutter"
-export "FLUTTER_APPLICATION_PATH=C:\Users\Shady\flutter_clean_architecture\example"
-export "FLUTTER_TARGET=lib\main.dart"
+export "FLUTTER_ROOT=/Users/rafaelcmm/SDKs/flutter"
```

Comment:
Is that correct? Should it be your computer's address?

Reply:
This is auto-generated by flutter and doesn't have any impact. It changes when other person runs the example.

Reply:
Should be ok. This file shouldn't be tracked anyway.